Sources/TSCUtility/PersistenceCache.swift

```diff
@@ -39,6 +39,10 @@ public final class SQLiteBackedPersistentCache: PersistentCacheProtocol {
     deinit {
         try? db.close()
```

Comment:
I wonder if this should call just `close()` now?  It's a final class, so I guess it doesn't matter, but seems like a more correct pattern in general.

Reply:
Arguably it should always have called `self.close()` — it's not a consequence of this particular diff, just more cleanup.

Reply:
good point